security/cmd/istio_ca/main.go

```diff
@@ -183,6 +188,14 @@ func init() {
 	flags.DurationVar(&opts.probeCheckInterval, "probe-check-interval", defaultProbeCheckInterval,
 		"Interval of checking the liveness of the CA.")
 
+	// Generate webhook secrets
+	flags.BoolVar(&opts.genWebhookSecrets, "generate-webhook-secrets", false,
```

Comment:
I don't think you need this boolean. Just use empty injector-webhook-secret to indicate it's disabled.